js-w2-hw.js

```diff
@@ -0,0 +1,129 @@
+// Flight booking fullname function
+
+function getFullname(firstName, lastName, useFormalName, gender) {
+  // Check if useFormalName is undefined
+  if (useFormalName === undefined) {
+    return "Error: Missing argument";
+  }
```

Comment:
useFormalName should not be a required argument. You could have assigned a default value as false for example:

```js
function getFullname(firstName, lastName, useFormalName = false, gender)
```